crmsh/ui_resource.py

```diff
@@ -293,6 +293,9 @@ def _commit_meta_attrs(self, context, resources, name, value):
 
         rc = True
         for rsc in resources:
+            rc, reason = xmlutil.RscState().is_managed(rsc)
+            if not rc:
+                context.error("Resource {} is unmanaged ({})".format(rsc, reason))
```

Comment:
Confusing terminology here, since "unmanaged" does not match to the fact when the "maintenance" mode is true.

Reply:
@zzhou1 For the display of `crm_mon`, both maintenance and is-manage=false situation, the resource status always show `unmanaged`